conftest.py

```diff
@@ -24,3 +30,32 @@ def pytest_addoption(parser):
         metavar='ticket',
         default=os.getenv('TICKET_NUM', None),
         help='Ticket number the tests should use')
+    parser.addoption(
+        "--env",
+        dest="env",
+        default="stage",
+        help="Environment tests are running in: dev | stage | prod"
+    )
+    parser.addoption(
+        "--api-version",
+        dest="apiversion",
+        default="0.0.0",
+        help="Version of the autopush service API we are testing against"
+    )
+
+
+@pytest.fixture()
+def env(request):
+    return request.config.getoption("--env")
+
+
+@pytest.fixture()
+def apiversion(request):
```

Comment:
per pep8, should be:  "api_version"
https://www.python.org/dev/peps/pep-0008/#function-names